changedetectionio/flask_app.py

```diff
@@ -542,13 +541,15 @@ def ajax_callback_send_notification_test(watch_uuid=None):
         from changedetectionio.apprise_plugin import apprise_custom_api_call_wrapper
         is_global_settings_form = request.args.get('mode', '') == 'global-settings'
         is_group_settings_form = request.args.get('mode', '') == 'group-settings'
-
         # Use an existing random one on the global/main settings form
-        if not watch_uuid and (is_global_settings_form or is_group_settings_form):
+        if not watch_uuid and (is_global_settings_form or is_group_settings_form) \
+                and len(list(datastore.data['watching'].keys())) > 0:
```

Comment:
```suggestion
                and datastore.data.get('watching'):
```

this is the same but easier/shorter